src/scripts/main.js

```diff
@@ -1 +1,21 @@
 'use strict';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const gallery = document.getElementById('thumbs');
+  const largeImg = document.getElementById('largeImg');
+
+  // eslint-disable-next-line no-shadow
+  gallery.addEventListener('click', (event) => {
+    event.preventDefault();
+
+    const target = event.target.closest('a');
```

Comment:
Consider checking if `event.target` is not null before calling `closest('a')` to avoid potential errors if the click event is triggered by a non-element node.